trade-finance-manager-ui/server/controllers/case/cancellation/reason-for-cancelling.controller.ts

```diff
@@ -3,19 +3,51 @@ import { CustomExpressRequest } from '@ukef/dtfs2-common';
 import { PRIMARY_NAVIGATION_KEYS } from '../../../constants';
 import { asUserSession } from '../../../helpers/express-session';
 import { ReasonForCancellingViewModel } from '../../../types/view-models';
+import { validateReasonForCancelling } from './validation/validate-reason-for-cancelling';
+
+export type GetReasonForCancellingRequest = CustomExpressRequest<{ params: { _id: string } }>;
+export type PostReasonForCancellingRequest = CustomExpressRequest<{ params: { _id: string }; reqBody: { 'reason-for-cancelling': string | undefined } }>;
```

Comment:
```suggestion
export type PostReasonForCancellingRequest = CustomExpressRequest<{ params: { _id: string }; reqBody: { 'reason': string} }>;
```

I like the custom Express request approach for an effective payload validation, but I think the body parameters should not allow `undefined` and can we keep them to pascal case please.

Reply:
Good shout @abhi-markan 

I think in all instances it will always be sent a string - if an empty form is submitted it will be an empty string